### PR TITLE
Replace Circe with uJson, add `CypherValue.toJson`

### LINKED ISSUE #485: Replace circe with uPickle in Zeppelin JsonSerialization

### PR DESCRIPTION
**Pull request by pstutz**
Fixes #485

### CODE REVIEW

**Comment by pstutz** [comment]
@DarthMax Does Zeppelin network visualisation support String IDs? Couldn't get a definitive answer in the documentation. Converting a `Long` to a JavaScript `Number` is not safe: [`JavaScript has a Number type which is a 64 bit floating point number`](https://stackoverflow.com/questions/5450012/how-to-convert-a-string-to-long-in-javascript?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa).

File: spark-cypher-testing/src/test/scala/org/opencypher/spark/web/CAPSJsonSerialiserTest.scala [hunk 1]
```diff
@@ -1,451 +0,0 @@
...
-import org.opencypher.okapi.ir.api.expr.Var
-import org.opencypher.okapi.relational.impl.syntax.RecordHeaderSyntax._
-import org.opencypher.okapi.relational.impl.table.{OpaqueField, RecordHeader}
-import org.opencypher.spark.impl.{CAPSGraph, CAPSRecords}
-import org.opencypher.spark.testing.CAPSTestSuite
-import org.opencypher.spark.testing.fixture.TeamDataFixture
-import org.opencypher.spark.web.CAPSJsonSerialiser.toJsonString
-
-//noinspection NameBooleanParameters
-class CAPSJsonSerialiserTest extends CAPSTestSuite with TeamDataFixture {
```
Comment (s1ck) [inline spark-cypher-testing/src/test/scala/org/opencypher/spark/web/CAPSJsonSerialiserTest.scala]: phew ... luckily it took me only ~30 seconds to fix those test in the RH PR :)

File: okapi-api/src/main/scala/org/opencypher/okapi/api/value/CypherValue.scala [hunk 2]
```diff
@@ -181,6 +185,33 @@ object CypherValue {
       }
     }
 
+    def toJson: Js.Value = {
```
Comment (s1ck) [inline okapi-api/src/main/scala/org/opencypher/okapi/api/value/CypherValue.scala]: nice!

**Comment by DarthMax** [comment]
Mhh I have no idea. For the visualization case it would not really have an implication if the reverse operation of to json is not safe

**Comment by pstutz** [comment]
We should probably have a more efficient Json table representation. Array of header names, then rows that are arrays.

**Comment by DarthMax** [comment]
I think it was used for the simple web service we had build as a show case once. 
I think we can keep it as it is. The strength of JSON is readability and accessibility not memory efficient storage.

**Comment by pstutz** [comment]
Removed `CypherRecords.toJson` after additional conversation with @DarthMax

**Pull request merged**